### PR TITLE
SITES-712 prevent commit of secrets to git repository

### PR DESCRIPTION
**Pull request by freshtonic**

### CODE REVIEW

**Comment by ltankey** [comment]
LGTM

**Comment by tessereth** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -56,6 +56,19 @@ $EDITOR .env
...
 ```
 
+### Prevent accidental commits of security credentials
+
+1. Install "git secrets"
+
+		$ brew install git-secrets
+		$ cd $YOUR_PROJECT_WORKDIR
+		$ git secrets --add-provider ./git-secret-patterns.rb
+		$ git secrets --register-aws
```
Comment (tessereth) [inline README.md]: missing `git secrets --install`

Comment (freshtonic) [inline README.md]: Addressed

**Comment by tessereth** [comment]
LGTM

**Pull request merged**